grunt_tasks/server.js

```diff
@@ -44,7 +44,8 @@ module.exports = function (grunt) {
     grunt.config.merge({
         shell: {
             readServerConfig: {
-                command: 'env python config_parse.py girder/conf/girder.local.cfg',
+                command: 'env python config_parse.py ' +
+                         grunt.config.get('girderDir') + '/conf/girder.local.cfg',
```

Comment:
I was hoping to use the static path for #1046 but also found this task to be a bit frustrating to work around. It would be nice if we could make them runtime parameters, but what does that mean for generically running grunt?